lib/tramway/engine.rb

```diff
@@ -14,6 +14,10 @@ class Engine < ::Rails::Engine
       configure_pagination if Tramway.config.pagination[:enabled]
     end
 
+    initializer 'your_gem.assets.precompile' do |app|
```

Comment:
generic name?

Reply:
You're right! Fixed :slightly_smiling_face: